agendas/2024/03-Mar/07-wg-primary.md

```diff
@@ -133,6 +133,8 @@ hold additional secondary meetings later in the month.
    - [#1032](https://github.com/graphql/graphql-spec/pull/1032) - define "selection set" (already has 2 TSC approvals)
    - [#894](https://github.com/graphql/graphql-spec/pull/894) - clarify "before execution starts" still includes the "request error"s that can be raised in `ExecuteRequest()` (variable coercion failure; not exactly one subscription field assertion failure)
    - [#1069](https://github.com/graphql/graphql-spec/pull/1069) - enforce consistent punctuation in the spec (with CI check)
+1. Strict error paths (5m, Martin)
+   - [#1073](https://github.com/graphql/graphql-spec/pull/1073) - Strict error paths 
```

Comment:
I put this towards the beginning because I think it shouldn't take too much time. Feel free to shuffle around if needed.